openapi/horizon/bundled.yml

```diff
@@ -1163,6 +1163,145 @@ paths:
               examples:
                 SubmitaTransaction:
                   $ref: '#/components/examples/SubmitaTransaction'
+  /transactions_async:
+    post:
+      summary: Submit Async Transaction
```

Comment:
Its longer, but maybe we should consider "Submit a Transaction Asynchronously". I think that better describes the approach, since the transaction itself isn't async, its the process for submitting it that is async.

Reply:
Yeah, I think I'm with @JakeUrban on that. Something that more clearly denotes the submission is the async part, not the tx. What about something like "Submit Transaction (Async)"?

Reply:
@JakeUrban @ElliotFriend Yeah I agree. I think "Submit a Transaction Asynchronously" sounds better here.